File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int GetCorrectInt();

int main(void) 
{

	int height = GetCorrectInt();
	int numHashes = 2;
	for (int i=0; i<height; i++) {
		for (int j=i; j<height-1; j++) {
			printf("%s", " ");
		}
		for (int k=0; k<numHashes; k++) {
			printf("%s", "#");
		}
		numHashes ++;
		
		printf("\n");
	}

}

int GetCorrectInt(void) 
{

	int height;
	do
	{
		printf("height: ");
		height = GetInt();
	}
	while (height < 0 || height > 23);
	return height;
}

File: pset6/mario.py
from cs50 import get_int

ready = False

while(not ready):
    
    print("Height: ", end="")
    
    height = get_int()
    
    if (height >= 0 and height <= 23):
        
        ready = True
        
for i in range(height):
    
    blank_space = height - i - 1
    
    print(" " * blank_space, end="")
    print("#" * ( i + 1 ), end="")
    print("  ", end="")
    print("#" * ( i + 1 ), end="")
    print(" " * blank_space)




